modules/cnf-performing-end-to-end-tests-running-in-single-node-cluster.adoc

```diff
@@ -24,21 +19,24 @@ When executing `podman` commands as a non-root or non-privileged user, mounting
 
 * You have logged in as a user with `cluster-admin` privileges.
 
+* You have applied a cluster performance profile by using Node Tuning Operator.
+
 .Procedure
 
 * To run the latency tests on a {sno} cluster, run the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e DISCOVERY_MODE=true -e FEATURES=performance -e ROLE_WORKER_CNF=master \
-registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
-/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+-e LATENCY_TEST_RUNTIME=<time in seconds> registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
+/usr/bin/test-run.sh --ginkgo.v
 ----
 +
 [NOTE]
 ====
-`ROLE_WORKER_CNF=master` is required because master is the only machine pool to which the node belongs. For more information about setting the required `MachineConfigPool` for the latency tests, see "Prerequisites for running latency tests".
+The default runtime for each test is 300 seconds. 
+For valid latency test results, run the tests for at least 12 hours by updating the variable `LATENCY_TEST_RUNTIME`.
```

Comment:
I'd suggest changing the word order - "by updating the `LATENCY_TEST_RUNTIME`  variable."

Reply:
corrected. Thanks